graphql_compiler/schema_transformation/subclass.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2017-present Kensho Technologies, LLC.
+from graphql.type.definition import GraphQLObjectType, GraphQLUnionType
+import six
+
+
+# NOTE: taken from bojan's macro system branch
+
+
+def _add_transitive_closure(graph):
```

Comment:
All of this is fine for now, but in the immediate future we may move away from type equivalence hints and toward schema graph, since that API is stabilized and available. It's fine to save that for the next PR, just giving you a heads-up.